templates/music/album_detail.html

```diff
@@ -4,6 +4,37 @@
 {% load markdown_deux_tags %}
 {% load music %}
 
+{% block socialmeta %}
+    <!-- Google+ data -->
+    <meta itemprop="name" content="{{ album.name }} by {{ artist.name|upper }}">
+    <meta itemprop="description" content="Discover and invest in music like {{ album.name }} on PerDiem.">
+    {% thumbnail album.photo.img "506" as thumb %}
+        <meta itemprop="image" content="{{ thumb.url }}">
+    {% endthumbnail %}
+    <!-- Twitter Card data -->
+    {% if artist.social_twitter %}
+        <meta name="twitter:card" content="summary_large_image">
+        <meta name="twitter:site" content="{{ artist.social_twitter.username_twitter }}">
```

Comment:
Everywhere that you've written `artist.` in tags should really be `album.project.artist.` on this page.